### ISSUE TITLE
Packer binary size increase (+40%) from 1.8.5 to 1.8.6

### ISSUE THREAD

**Issue by Fredouye** [hashicorp/packer#12392]
#### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

When filing a bug, please include the following headings if possible. Any
example text in this template can be deleted.

#### Overview of the Issue

Hi !

Packer binary went from 145 Mb (1.8.5) to 205 Mb (1.8.6), I haven't seen anything in the Changelog about added "code".

#### Reproduction Steps

```
# docker pull hashicorp/packer:light-1.8.5
# docker pull hashicorp/packer:light-1.8.6
# docker image ls
REPOSITORY                       TAG                          IMAGE ID       CREATED          SIZE
hashicorp/packer                 light-1.8.6                  cdaf1dad0305   2 months ago     240MB
hashicorp/packer                 light-1.8.5                  e0e7f484d23d   4 months ago     181MB
```

```
# docker run --rm --entrypoint "/bin/ls" hashicorp/packer:light-1.8.6 -l /bin/packer
-rwxr-xr-x    1 root     root     204974532 Feb 15 15:42 /bin/packer

# docker run --rm --entrypoint "/bin/ls" hashicorp/packer:light-1.8.5 -l /bin/packer
-rwxr-xr-x    1 root     root     145444864 Dec 12 21:50 /bin/packer
```

### Packer version

1.8.6

### Simplified Packer Template

N/A

### Operating system and Environment details

Docker 23.0.5

### Log Fragments and crash.log files

N/A

**Comment by nywilken** [comment]
Thanks for bubbling this up. We will take a look to see what might've caused this increase in size.

**Comment by nywilken** [comment]
Hi @Fredouye thanks again for bubbling this up. I took some time to look into the size for the binaries and found that the last two releases did not strip the debug symbol tables from the binaries. This is why the binaries are larger than v1.8.5 with no additional code changes. 

Right before 1.8.6 there was a change to our build pipeline for injecting version information into the release binaries. I didn't notice that the flags for removing the symbol tables were removed. I opened a PR to fix this for the next release.
